src/main/java/run/halo/oauth/Oauth2LoginConfiguration.java

```diff
@@ -61,8 +64,9 @@
  */
 @Slf4j
 @Getter
-@Component
-public final class Oauth2LoginConfiguration {
+@Configuration
+@EnableAsync
+public class Oauth2LoginConfiguration {
```

Comment:
Why do I remove the final keyword?

Because the annotation `@Configuration` requires the class is non-final.